### ISSUE TITLE
Uninstall option with remove user data leaves data in AppData

### ISSUE THREAD

**Issue by senpl** [RocketChat/Rocket.Chat.Electron#2775]
### Search before asking

- [X] I had searched in the [issues](https://github.com/RocketChat/Rocket.Chat.Electron/issues?q=is%3Aissue) and found no similar issues.

### Operating System

- [ ] macOS
- [X] Windows
- [ ] Linux

### Operating System Version

Windows 11

### It happens on the web browser too?

No, it just happens on the Desktop app

### Rocket.Chat Desktop App Version

3.9.10

### Rocket.Chat Server Version

latest (anyway not related)

### Describe the bug

Uninstall not removes user data. User has to remove data from his folders.

### How to Reproduce

1. Install rocket.chat.
2. login to some server
3. Close rocket.chat
4. Uninstall from uninstall.exe
5. Uninstaller check if remove user data (we click yes) so field is checked.
6. Data from folders users\youruser\AppData\Roaming and users\youruser\AppData\Local are not removed. Probably should be.

### Describe your Expected behavior

This folder should be deleted

### Are you willing to submit a code contribution?

- [ ] Yes, I am willing to submit a Pull Request!

**Comment by Sumitwarrior7** [comment]
I'm interested in addressing this bug. Could you kindly assign the issue to me

**Comment by GuptaPrisha** [comment]
Subject: Acknowledgment and Request for Assignment

Hi @jeanfbrito,

I appreciate @senpl 's keen observation in identifying a significant flaw in user experience and system functionality. I would like to express gratitude for bringing attention to this issue. Considering its importance, I'm eager to take on the responsibility of addressing and resolving this in upcoming releases.

Thank you,
Prisha

**Comment by GuptaPrisha** [comment]
Hi @jeanfbrito,

I appreciate @senpl 's keen observation in identifying a significant flaw in user experience and system functionality. I would like to express gratitude for bringing attention to this issue. Considering its importance, I'm eager to take on the responsibility of addressing and resolving this in upcoming releases.

Thank you,
Prisha

**Comment by preeesha** [comment]
Heyy @senpl which version of Rocket.Chat desktop are you using? Development or Production?

Also, can you please check if there's any other folder with "Rocket.Chat" in its name in the `%APPDATA%` (your user) directory.

Let me know.

**Comment by preeesha** [comment]
After researching on the issue I realized that it only happens in the debug version of the app. In the debug version, it creates a folder named `Rocket.Chat (development)` rather than just `Rocket.Chat` which installer obviously fails to remove as it searches for just `Rocket.Chat`.

This issue is not there in the production builds of the application.

@jeanfbrito I would like you to have a look at this and preferably close the issue.

Thanks
Prisha Gupta

**Comment by senpl** [comment]
> Heyy @senpl which version of Rocket.Chat desktop are you using? Development or Production?
> Also, can you please check if there's any other folder with "Rocket.Chat" in its name in the `%APPDATA%` (your user) directory.
> Let me know.

Yes. I probably used version from github. I already deleted that folders so I am no longer have them. Still it was probably Development version, but all app version (Also from Microsoft store) used that data.

**Comment by FlyingT** [comment]
Still an issue
